src/classes/handlers/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class ContactTriggerHandler {
+
+    public static void handler(List<Contact> newTrigger,  Map<Id,Contact> oldMapContacts, Map<Id,Contact> newMapContacts, System.TriggerOperation triggerOperation){
+        switch on TriggerOperation {
+            when AFTER_INSERT{
+                ContactService.createTeamMember(newTrigger);
```

Comment:
MINOR
ContactService reference here is optional as methods could be referenced in the handler class as private, but it`s fine to use service as well.

Reply:
ok